package.json

```diff
@@ -7,6 +7,7 @@
   },
   "dependencies": {
     "async": "0.2.7",
+    "aws-sdk": "^2.2.4",
```

Comment:
I think that the aws-sdk and nodegit dependencies should be in the dependencies list